src/pykeen/models/unimodal/structured_embedding.py

```diff
@@ -70,112 +71,33 @@ def __init__(
         :param kwargs:
             Remaining keyword arguments to forward to :class:`pykeen.models.EntityEmbeddingModel`
         """
+        # Embeddings
+        init_bound = 6 / np.sqrt(embedding_dim)
```

Comment:
This seems to be equivalent to the pykeen variant of xavier-uniform: https://github.com/pykeen/pykeen/blob/master/src/pykeen/nn/init.py#L26

Reply:
Indeed. fixed in 5338da14